.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: c
```

Comment:
I suggest to use Azure Pipelines instead: https://github.com/xerpi/libvita2d/pull/78

Reply:
Why azure pipelines is better than Travis ?

Reply:
Native Docker support, built-in artifact support, better UI/UX in general, and the fact that Travis-CI as a product has been put in maintainance mode: see https://news.ycombinator.com/item?id=18978346 and https://news.ycombinator.com/item?id=19256970.